hooks/tk-multi-publish2/basic/collector.py

```diff
@@ -53,8 +53,10 @@ def process_current_session(self, parent_item):
             )
 
             self.collect_playblasts(item, project_root)
-            self.collect_alembic_caches(item, project_root)
-
+            if cmds.ls(geometry=True, noIntermediate=True):
+                self.collect_alembic_caches_geometry(item)
```

Comment:
minor: thinking about this a little bit, and I think we should call this something like `collect_session_geometry`. The existing `collect_alembic_caches` is collecting already exported `.abc` files. For what you're doing, we're collecting geometry in the session that we're going to export.